app/controllers/users_controller.rb

```diff
@@ -18,6 +24,27 @@ def create
     end
   end
 
+  def update
+    @user = User.find_by(id: params[:id])
+    if @user.update_attributes(user_params)
+      flash[:success] = "Profile updated"
+      redirect_to @user
+    else
+      render "edit"
+    end
+  end
+
+  def destroy
```

Comment:
With action destroy user, you need to check admin permission for user
Please refer the sample app for this

Reply:
is it  need this  before_action :admin_user, only: :destroy to allow admin to log in first then destroy?